src/main/java/org/commcare/core/graph/c3/AxisConfiguration.java

```diff
@@ -118,10 +121,21 @@ private void addTickConfig(JSONObject axis, String key, String varName) throws G
                         } else {
                             values.put(parseDouble(location, key));
                         }
+                        try {
+                            String current = labels.getString(location);
+                            if (current.length() > largestLabel.length()) {
+                                largestLabel = current;
+                            }
+                        } catch (JSONException e) { }

```

Comment:
should we throw an error here ?

Reply:
Nope, the graphs work perfectly fine as of now. Just that the labels can overlap. Here, I'm trying to get the largest label length, and if `getString` throws an exception, then it doesn't matter, we simply won't rotate the label.

Reply:
makes sense, thanks